packages/cxl-lumo-styles/scss/themes/vaadin-context-menu-list-box.scss

```diff
@@ -12,3 +12,25 @@
     box-shadow: inset 0 0 0 ($vaadin-focus-ring-thickness * 0.5) var(--lumo-primary-color);
   }
 }
+
+:host([theme~="cxl-marketing-nav"]) {
+  box-shadow: none !important; /* stylelint-disable-line declaration-no-important */
+  // box-shadow: 0 0 0 1px var(--lumo-shade-5pct), var(--lumo-box-shadow-m);
```

Comment:
I assume this is work in progress, but just wanted to highlight we should not leave commented out rules in the final code.